eyeloop/utilities/parser.py

```diff
@@ -59,7 +59,7 @@ def to_csv(self):
         try:
             import pandas as pd
         except:
-            print("Please make sure that pandas is installed (pip install pandas).")
+            raise Exception("Please make sure that pandas is installed (pip install pandas).")
```

Comment:
Raising errors so the execution stops, a traceback is printed, and the code stops. Otherwise it will continue, try to access undefined variables, or use modules that were not imported, resulting in errors that may be hidden from the user.